src/components/title/edit/title-edit.js

```diff
@@ -123,12 +124,19 @@ class TitleEdit extends Component {
                 label="Title information"
               >
                 <NameField />
+
+                <ContributorField
+                  initialValue={initialValues.contributors}
+                />
+
                 <EditionField />
                 <PublisherNameField />
                 <PublicationTypeField />
-                <ContributorField
-                  initialValue={initialValues.contributors}
+
+                <IdentifiersFields
+                  initialValue={initialValues.identifiers}
```

Comment:
Not a blocker by any means, but just a question: Why do we need `initialValue` here? I thought `redux-form` did that for us when we pass `initialValues` to the form.

Reply:
@wwilsman it's a weird strategy for this use case:
If I have a previously saved identifier, open the edit route, and clear all the identifier rows, I get a message warning me that saving will remove all the identifiers. There's probably a more elegant way to calculate that state.

Reply:
Ok, intersting. I thought `redux-form` was mapping the props just like this behind the scenes, but who knows what it actually does. Lol